### PR TITLE
doc: simplify archive policy example, fix worst case scenario

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: doc/source/operating.rst [hunk 1]
```diff
@@ -66,29 +66,32 @@ seconds: there might be a gap if data is fed irregularly.
...
-you use the 8 standard |aggregation method|, your |metric| will take up to
-8 × 54 KiB = 432 KiB of disk space.
+The worst case scenario for storing compressed data points is 8.04 bytes per
+point, whereas best case scenario can compress up to 0.05 bytes per point.
+Knowing that, it is possible to compute the worst case scenario for storage in
+order to plan for data storage capacity.
+
+An archive policy of 1440 points would need 1440 points × 8.04 bytes = 11.3 KiB
+per |aggregation method|. If you use the 8 standard |aggregation method|
+propsed by Gnocchi, your |metric| will take up to 8 × 11.3 KiB = 90.42 KiB of
```
Comment (chungg) [inline doc/source/operating.rst]: s/8/6/

Comment (jd) [reply doc/source/operating.rst]: hm why 6? it's 8 agg methods

Comment (chungg) [reply doc/source/operating.rst]: but you list six above? *mean*, *min*, *max*, *sum*, *std*, *count*. i think it was 8 in v3.

Comment (jd) [reply doc/source/operating.rst]: Holy cow, you're right. So I'm double fixing stuff.

**Comment by jd** [comment]
Pull-request updated, HEAD is now ccd40a247b30684e95c7e68e24f20d9a4e99b661

File: doc/source/operating.rst [hunk 2]
```diff
@@ -66,29 +66,32 @@ seconds: there might be a gap if data is fed irregularly.
...
-you use the 8 standard |aggregation method|, your |metric| will take up to
-8 × 54 KiB = 432 KiB of disk space.
+The worst case scenario for storing compressed data points is 8.04 bytes per
+point, whereas best case scenario can compress up to 0.05 bytes per point.
+Knowing that, it is possible to compute the worst case scenario for storage in
+order to plan for data storage capacity.
+
+An archive policy of 1440 points would need 1440 points × 8.04 bytes = 11.3 KiB
+per |aggregation method|. If you use the 6 standard |aggregation method|
+propsed by Gnocchi, your |metric| will take up to 6 × 11.3 KiB = 67.8 KiB of
```
Comment (chungg) [inline doc/source/operating.rst]: s/propsed/proposed/

**Comment by jd** [comment]
Pull-request updated, HEAD is now 36604431b81f5edb0bc8636d700a9e236645ef89

**Pull request merged**